### PR TITLE
Align Names of Projects, Packages and Classes

### LINKED ISSUE #2221: Ensure package names are aligned for various clients

### LINKED ISSUE #2228: Rename `EventStoreGrpcClient` to `EventStoreClient`
Since gRPC is the future of client protocols, we should not embed the protocol in the name. It is already in the namespace, and if people need to use both they can use alias imports.

### PR DESCRIPTION
**Pull request by thefringeninja**
Fixes #2228
Fixes #2221

### CODE REVIEW

File: src/EventStore.Client/EventStore.Client.csproj [hunk 1]
```diff
@@ -33,7 +34,7 @@
 	</ItemGroup>
 	<ItemGroup>
 		<Protobuf Include="../Protos/Grpc/*.proto" Exclude="../Protos/Grpc/operations.proto" Access="internal" GrpcServices="Client" ProtoRoot="../Protos/Grpc" />
-		<Compile Include="../EventStore.Grpc.Common/*.cs" />
+		<Compile Include="../EventStore.Client.Common/*.cs" />
 	</ItemGroup>
 	<ItemGroup>
 		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```
Comment (pgermishuys) [inline src/EventStore.Client/EventStore.Client.csproj]: We missed the InternalsVisible to Attribute here.
We can remove the `AssemblyInfo.cs` and just add the necessary attributes inline in the csproj.

```
	<ItemGroup>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.Tests</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.Operations</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.Projections.Tests</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.UserManagement.Tests</_Parameter1>
		</AssemblyAttribute>
		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
			<_Parameter1>EventStore.Client.PersistentSubscriptions.Tests</_Parameter1>
		</AssemblyAttribute>
	</ItemGroup>
```

File: src/EventStore.Client/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("EventStore.Client.Tests")]
```
Comment (pgermishuys) [inline src/EventStore.Client/Properties/AssemblyInfo.cs]: We can safely remove this file. See the comment on `EventStore.Client.csproj`

**Comment by jen20** [comment]
This looks good to me pending CI.

File: src/EventStore.Client.Operations/EventStore.Client.Operations.csproj [hunk 3]
```diff
@@ -15,6 +15,7 @@
 				<PackageReleaseNotes>https://eventstore.com/blog/</PackageReleaseNotes>
 				<Copyright>Copyright 2012-2019 Event Store Ltd</Copyright>
```
Comment (bartelink) [inline src/EventStore.Client.Operations/EventStore.Client.Operations.csproj]: 2020 ;)

**Pull request merged**